Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 # Update StackStorm version here to rely on other Docker images tags
 appVersion: 3.0dev
 name: stackstorm-ha
-version: 0.9.0
+version: 0.10.0
```

Comment:
Ideally version needs to be bumped just before the release, as by the fact this is not yet `0.10.0`.

Reply:
Oops, this slipped in there.. agreed.